### ISSUE TITLE
Request from Austin: feature tracker for instrumentation

### ISSUE THREAD

**Issue by avillela** [open-telemetry/sig-end-user#13]
Per request from @austinlparker in this [Slack conversation reference](https://cloud-native.slack.com/archives/C065JSPJN15/p1712858965286529) -- Tackle after #12

**Comment by danielgblanco** [comment]
Closing as duplicate of https://github.com/open-telemetry/sig-end-user/issues/12
